From 0905e94e48b6ce15598c2a32b4aff5d1410e4d82 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Fri, 12 Feb 2021 11:54:14 +0800
Subject: [PATCH] treepath.hg: Export comparison operators

The 'GTKMM_API' decoration was missed from the '_WRAP_COMPARE' item,
making the code fail to link for Inkscape on Windows.  Add the needed
decoration to fix this.

Fixes issue #88.
---
 gtk/src/treepath.hg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/src/treepath.hg b/gtk/src/treepath.hg
index e18e0f92..e22e9fea 100644
--- a/gtk/src/treepath.hg
+++ b/gtk/src/treepath.hg
@@ -162,7 +162,7 @@ _DEPRECATE_IFDEF_END
   _WRAP_METHOD(Glib::ustring to_string() const, gtk_tree_path_to_string)
 
 #m4begin
-  _WRAP_COMPARE(gtk_tree_path_compare)
+  _WRAP_COMPARE(gtk_tree_path_compare, , GTKMM_API)
 #m4end
 
   _IGNORE(gtk_tree_path_append_index) //replaced by push_back()
-- 
GitLab

